### PR TITLE
Update README.md to switch to the latest Miranj Logo

### PR DESCRIPTION
**Pull request by rishifter**
Updated the logo url to the same one used on the website.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<img align="right" src="https://cdn.miranj.in/mc3/img/apple-touch-icon-precomposed.png" width="57" height="57" alt="Miranj">
+<img align="right" src="https://cdn.miranj.in/v4/favicons/apple-touch-icon.png" width="57" height="57" alt="Miranj">
```
Comment (rungta) [inline README.md]: Incorrect logo version. Use the version that is used in, say, our Craft Partners profile. https://craftcms.com/partners/miranj

**Review by rungta — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rishifter) [reply README.md]: Updated from the Craft Partners Profile. 

Note: The image is square but on the Craft site the styles make it rounded.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<img align="right" src="https://cdn.miranj.in/mc3/img/apple-touch-icon-precomposed.png" width="57" height="57" alt="Miranj">
+<img align="right" src="https://cdn.craft.cloud/032c8560-18a8-45ba-a958-a03804e5cb66/assets/partners/1146942.png" width="57" height="57" alt="Miranj">
```
Comment (rungta) [inline README.md]: We should not depend on a file hosted on a third party server. That is the right image but the URL for this should be one that we have control over.

**Review by rungta — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (rishifter) [reply README.md]: Is there a place with a list of all branding resources with URLs from our CDN?

**Comment by rishifter** [comment]
Updated with image from Github profile. Assuming Github here is not considered third-party since the code is already hosted on their servers.

**Review by rungta — APPROVED** [review]

**Comment by rungta** [comment]
> Assuming Github here is not considered third-party since the code is already hosted on their servers.

Kind-of, but not entirely.
1. The Craft CDN URL was not a predictable API. It seemed like a computer generated URL that might change or stop working anytime Craft changes their code.
2. It is not something that we control in the sense that if we had to update our logo, we cannot impact that exact URL.
3. We would be leeching off bandwidth from someone else’s server, which is not only frowned upon, might even be explicitly disabled using CORS headers by Craft.
4. The GitHub URL looks like a predictable API which is tied to the organisation’s ID (which is not going to change unless we delete the org and re-create a new one).
5. I expect that even if we update our avatar on GitHub, the URL will remain the same.
6. We are using a GitHub hosted file on GitHub.com so we are staying within the same ecosystem.

**Pull request merged**